### PR TITLE
Add gmkb keyboard

### PR DESCRIPTION
**Pull request by diana202402**
## Description

Add gmkb keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
You seem to be attempting to add this as a personal project, rather than alongside the existing `keyboards/doio/` keyboards?

File: keyboards/diana/gmkb/keyboard.json
Comment (fauxpark) [inline keyboards/diana/gmkb/keyboard.json:1]: Please run `qmk format-json -i` on this file, and rename it to `keyboard.json`.

File: keyboards/diana/gmkb/rules.mk
Comment (fauxpark) [inline keyboards/diana/gmkb/rules.mk:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by diana202402** [comment]
> You seem to be attempting to add this as a personal project, rather than alongside the existing `keyboards/doio/` keyboards?

@zvecr  Hi, I can't revert the  files  I previously modified, 
              So I used a new personal account to update the code.

File: keyboards/diana/gmkb/rules.mk
Comment (diana202402) [reply keyboards/diana/gmkb/rules.mk:1]: @fauxpark  Hi, I've deleted it and formate the keyboard  file.

**Comment by zvecr** [comment]
> Hi, I can't revert the files I previously modified,
So I used a new personal account to update the code.

That doesn't make any sense.

It would need to be `keyboards/doio/nindoio` (or something similar). Anything associated to your accounts username is invalid.

**Comment by diana202402** [comment]
> > Hi, I can't revert the files I previously modified,
> > So I used a new personal account to update the code.
> 
> That doesn't make any sense.
> 
> It would need to be `keyboards/doio/nindoio` (or something similar). Anything associated to your accounts username is invalid.

@zvecr  Hi ,if I 

> > Hi, I can't revert the files I previously modified,
> > So I used a new personal account to update the code.
> 
> That doesn't make any sense.
> 
> It would need to be `keyboards/doio/nindoio` (or something similar). Anything associated to your accounts username is invalid.

@zvecr  HI， I have reverted the pull request as DOIO account.I clesed this pull request.  Thank you for your help.

**Pull request closed without merging**